@shared/api/internal.ts

```diff
@@ -437,13 +437,17 @@ export const signFreighterTransaction = async ({
 
 export const submitFreighterTransaction = async ({
   signedXDR,
-  networkUrl,
+  networkDetails,
 }: {
   signedXDR: string;
```

Comment:
ran into some funky typescript stuff, it looks like we've been passing the transaction object to this method to submit, instead of the xdr string. Not sure how Typescript allowed this before 🤷 , but changed it to pass the string and build the object here

Reply:
I had originally commented "why not just change the type?" but looks like it makes the ledger flow a little easier to pass in an xdr string here instead 👍